### PR TITLE
Added .env files to the react project templates in order to run the CRA server independently.

### LINKED ISSUE #7286: dotnet new react requires .env
_From @Rick-Anderson on February 1, 2019 21:38_

See https://github.com/aspnet/Docs/pull/10423 and [Use the React project template](https://docs.microsoft.com/en-us/aspnet/core/client-side/spa/react?view=aspnetcore-2.2&tabs=visual-studio)

Add a *.env* file to the *ClientApp* subdirectory with the following setting:

     `BROWSER=none`

This will prevent your web browser from opening when starting the CRA server externally.

_Copied from original issue: dotnet/templating#1752_

### PR DESCRIPTION
**Pull request by dannyBies**
Added .env files to the react project templates in order to run the CRA server independently.

PR based on #12435

Fixes #7286

### CODE REVIEW

File: src/Middleware/NodeServices/src/HostingModels/NodeInvocationException.cs [hunk 1]
```diff
@@ -56,4 +56,4 @@ public NodeInvocationException(string message, string details, bool nodeInstance
             AllowConnectionDraining = allowConnectionDraining;
         }
     }
-}
+}
```
Comment (mkArtakMSFT) [inline src/Middleware/NodeServices/src/HostingModels/NodeInvocationException.cs]: Seems like this file should be removed from this PR, as there is no meaningful change here.

Comment (dannyBies) [reply src/Middleware/NodeServices/src/HostingModels/NodeInvocationException.cs]: Not sure how that got in. I've reverted the changes made to this file.

**Review by mkArtakMSFT — APPROVED** [review]
Thanks @dannyBies.
@danroth27 given this is change on the template level, would be great to have your signoff on this too.

**Review by danroth27 — APPROVED** [review]
Seems reasonable.

**Review by ryanbrandenburg — APPROVED** [review]

**Comment by mkArtakMSFT** [comment]
Thanks for the PR, @dannyBies.

**Pull request merged**